### PR TITLE
Update documentation of glif models

### LINKED ISSUE #1782: Update glif_psc documentation
Currently, the glif_psc documentation does not render properly on Read the Docs: https://nest-simulator.readthedocs.io/en/latest/models/glif_psc.html

The "Parameters" and "References" sections need to be updated.

### PR DESCRIPTION
**Pull request by sarakonradi**
Currently, ``glif_cond.h`` and ``glif_psc.h`` do not render properly on Read the Docs. 

This PR updates the RST of these files, including:
- References
- Tables
- Bullet points
- Links
- See also

It fixes #1782.

### CODE REVIEW

File: models/glif_psc.h [hunk 1]
```diff
@@ -43,88 +43,48 @@ Description
...
 GLIF model mechanism setting is based on three parameters
 (spike_dependent_threshold, after_spike_currents, adapting_threshold).
 The settings of these three parameters for the five GLIF models are list below.
 Other combinations of these parameters will not be supported.
-GLIF Model 1 (LIF) - (False, False, False),
-GLIF Model 2 (LIF_R) - (True, False, False),
-GLIF Model 3 (LIF_ASC) - (False, True, False),
-GLIF Model 4 (LIF_R_ASC) - (True, True, False),
-GLIF Model 5 (LIF_R_ASC-A) - (True, True, True).
+
```
Comment (clinssen) [inline models/glif_psc.h:72]: Would a table be appropriate here?

File: models/glif_cond.h [hunk 2]
```diff
@@ -53,47 +53,48 @@ Description
...
-with units being converted from SI units (i.e., V, S (1/Ohm), F, s, A) to
-NEST used units (i.e., mV, nS (1/GOhm), pF, ms, pA) and values being rounded to
-appropriate digits for simplification.
+can be found and downloaded in the `Allen Cell Type Database
+<https://celltypes.brain-map.org>`_. For example, the default parameter setting of this
+glif_cond neuron model was from the parameter values of GLIF Model 5 of Cell
+490626718, which can be retrieved from the `Allen Brain Atlas
+<https://celltypes.brain-map.org/mouse/experiment/electrophysiology/
+490626718>`_, with units being converted from SI units (i.e., V, S (1/Ohm),
+F, s, A) to NEST used units (i.e., mV, nS (1/GOhm), pF, ms, pA) and values
```
Comment (terhorstd) [inline models/glif_cond.h:96]:
```suggestion
F, s, A) to NEST used units (i.e., mV, nS (1/GOhm), pF, ms, pA) and values
```

File: models/glif_psc.h (hunk 1 above)
Comment (sarakonradi) [reply models/glif_psc.h]: Done

**Review by clinssen — APPROVED** [review]
Thanks for the fast updates!

**Comment by sarakonradi** [comment]
@terhorstd Could you please have a final look too? Thanks!

**Review by terhorstd — APPROVED** [review]
LGTM

**Pull request merged**